galasa-ui/src/tests/routes/authCallback.test.ts

```diff
@@ -1,8 +1,12 @@
-/**
+/*
  * Copyright contributors to the Galasa project
- * @jest-environment node
+ *
+ * SPDX-License-Identifier: EPL-2.0
  */
 
+/* 
+* @jest-environment node
```

Comment:
I'm wondering if the `@jest-environment node` not appearing in the first comment block in the file is going to cause a problem.

Reply:
I don't know this stiff well enough.

Reply:
@KirbyKatcher Is this actually needed ? https://jestjs.io/docs/configuration says that 
this kind of thing can be put into a configuration file rather than in each source file ?

And will having it in the second comment block in a file cause a problem ?

Reply:
It's used to map the test files for jest. I have tested again, with the changes onto another comment and everything is passing as expected

Reply:
These files have this `@jest-environment node` annotation to override the environment in which these tests run (we've used `jest-environment-jsdom` in Jest's config file to simulate a browser environment, allowing for front-end components to be mounted in unit tests which the `node` environment does not support).

Since we'll be moving the webui's interactions with Dex into the API server's auth service very soon, the routes that these tests correspond to will not be needed for much longer and these annotations also won't be needed. For now, moving these annotations into the first comment block as they were previously should be enough to get the builds passing (see below for an example).

```ts
/*
 * Copyright contributors to the Galasa project
 *
 * SPDX-License-Identifier: EPL-2.0
 * 
 * @jest-environment node
 */
```

Reply:
thanks, I will make that change